### ISSUE TITLE
Cloudflare pages deploy error: Cannot access 'da' before initialization

### ISSUE THREAD

**Issue by luizzappa** [nuxt/nuxt#29767]
### Environment

- Operating System: Windows_NT
- Node Version:     v20.17.0
- Nuxt Version:     3.13.2
- CLI Version:      3.14.0
- Nitro Version:    2.10.1
- Package Manager:  npm@10.8.2
- Builder:          -
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -

### Reproduction

-

### Describe the bug

When updating the version from nuxt `3.8.1` to `3.13.2`, this error started to appear when deploying to cloudflare pages (Worker):

```
20:31:46.397 | ✨ Success! Uploaded 3 files (207 already uploaded) (0.92 sec)
-- | --
20:31:46.397 |  
20:31:46.659 | ✨ Upload complete!
20:31:49.337 | Success: Assets published!
20:31:51.011 | Error: Failed to publish your Function. Got error: Uncaught ReferenceError: Cannot access 'da' before initialization   at chunks/nitro/nitro.mjs:1:522814 in useRuntimeConfig   at chunks/nitro/nitro.mjs:1:386806
```

**Comment by luizzappa** [comment]
Ok, the Nitro `2.10.1` version resolved the problem, but the following message began to appear multiple times when trying to deploy to cloudflare:

```
10:02:28.640	[plugin commonjs--resolver] It appears a plugin has implemented a "resolveId" hook that uses "this.resolve" without forwarding the third "options" parameter of "resolveId". This is problematic as it can lead to wrong module resolutions especially for the node-resolve plugin and in certain cases cause early exit errors for the commonjs plugin.
10:02:28.641	In rare cases, this warning can appear if the same file is both imported and required from the same mixed ES/CommonJS module, in which case it can be ignored.
```

My `package.json`

```
 "overrides": {
    "vue": "^3.5.12",
    "nitropack": "2.10.1"
  }
```

**Comment by pi0** [comment]
Please try recreating your lockfile. If issue persists sharing a reproduction can be super helpful to investigate.

**Comment by pi0** [comment]
Warning should be solved in latest Nitro (2.10.2)

**Comment by luizzappa** [comment]
I updated to version `2.10.2` of Nitro and the messages no longer appear. Thanks for the quick support @pi0 !
